src/pages/Home/subcomponents/FriendReadonlyCountry.js

```diff
@@ -4,7 +4,7 @@ import {
   ComposableMap,
   ZoomableGroup,
   Geographies,
-  Geography
+  Geography,
```

Comment:
@GreenWithMV definitely very minor, but I believe the preferred style here is to not have a trailing comma on multiple imports.